### PR TITLE
React tables

### PR DESCRIPTION
**Pull request by datazuul**
* react tables for geolocations and corporatebodies
* backend for getting languages of identifiables
* README.md updated

### CODE REVIEW

File: dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/agent/CorporateBodiesController.java [hunk 1]
```diff
@@ -87,19 +85,33 @@ public String edit(
...
+  @ResponseBody
+  public PageResponse<CorporateBody> findAll(
+      @RequestParam(name = "pageNumber", required = false, defaultValue = "0") int pageNumber,
+      @RequestParam(name = "pageSize", required = false, defaultValue = "25") int pageSize)
+      throws HttpException {
+    List<Order> orders = new ArrayList<>();
+    Order labelOrder = new Order("label");
+    labelOrder.setSubProperty(localeService.getDefaultLanguage().getLanguage());
+    orders.addAll(Arrays.asList(labelOrder));
+    Sorting sorting = new Sorting(orders);
```
Comment (morpheus-87) [inline dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/agent/CorporateBodiesController.java:98]: Braucht es diese Sortierung wirklich?

File: dc-cudami-editor/src/locales/en/translation.json [hunk 2]
```diff
@@ -51,6 +51,8 @@
       "digitalObjectFromProject": "Are you sure that you want to remove this digital object from the project?",
       "subcollectionFromCollection": "Are you sure that you want to remove this sub collection from the collection?"
     },
+    "corporateBody": "Corporate Body",
```
Comment (morpheus-87) [inline dc-cudami-editor/src/locales/en/translation.json:54]:
```suggestion
    "corporateBody": "Corporate body",
```

File: dc-cudami-editor/src/locales/en/translation.json [hunk 3]
```diff
@@ -51,6 +51,8 @@
...
+    "corporateBodys": "Corporate Bodies",
```
Comment (morpheus-87) [inline dc-cudami-editor/src/locales/en/translation.json:55]:
```suggestion
    "corporateBodys": "Corporate bodies",
```

**Review by morpheus-87 — CHANGES_REQUESTED** [review]

File: dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/agent/CorporateBodiesController.java (hunk 1 above)
Comment (datazuul) [reply dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/agent/CorporateBodiesController.java]: done

**Review by morpheus-87 — APPROVED** [review]

**Pull request merged**